client/app/views/contacts/card.coffee

```diff
@@ -64,32 +57,28 @@ module.exports = class ContactCardView extends Mn.LayoutView
 
 
     serializeData: ->
-        _.extend super,
-            lists: CONFIG.datapoints.types
-            fullname: @model.toString pre: '<b>', post: '</b>'
+        _.extend {}, super, lists: CONFIG.datapoints.types
 
 
-    onRender: ->
-        @$('.datapoints').each (index, el) =>
-            name = el.dataset.type
-            @addRegion name, "[data-type=#{name}]"
-            @showChildView name, new DatapointsView
-                collection:    @model[name]
-                name:          name
-                cardViewModel: @model
+    _deleteBehavior: ->
+        opts = name: @options.model.get 'fn'
 
-        @addRegion 'xtras-infos', '[data-type=xtras-infos]'
-        @showChildView 'xtras-infos', new XtrasView model: @model
+        event:      'delete'
```

Comment:
I'm not a big of declaring useless `var` in scope, but can it be a better method name, like `_getDeleteModalCfg`?